### PR TITLE
Add 60 seconds to waiting time, use tqdm for logger

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
None

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements

Adds 60 seconds to the API rate limit reset waiting time. This is because previously it could happen that the `mi` was faster than reset time and the analysis failed

Use of `tqdm` module to use with analysis (instead of manually printing it to the logger), see screenshot of a tiny little progressbar:
image

### CODE REVIEW

File: srcopsmetrics/iterator.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def run(self):
                     wait_time = gh_time - local_time.replace(tzinfo=None)
```
Comment (KPostOffice) [inline srcopsmetrics/iterator.py:84]: I think it makes a little more sense to add the 60 seconds here instead.

Comment (pacospace) [reply srcopsmetrics/iterator.py:84]: Also might be good to have an env variable to tune that time if you want?

**Pull request merged**